lang/cs/Org.Apache.REEF.All/packages.config

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It is not needed in All.csproj and Jar.csproj. But if we move all the scripts to build.prop, we might need to exclude it from those projects. Or leave the default as there is not harm.